src/dxvk/dxvk_image.h

```diff
@@ -251,8 +251,13 @@ namespace dxvk {
      * \returns A compatible image layout
      */
     VkImageLayout pickLayout(VkImageLayout layout) const {
-      return m_info.layout == VK_IMAGE_LAYOUT_GENERAL
-        ? VK_IMAGE_LAYOUT_GENERAL : layout;
+      if (unlikely(m_info.layout == VK_IMAGE_LAYOUT_ATTACHMENT_FEEDBACK_LOOP_OPTIMAL_EXT))
+        return VK_IMAGE_LAYOUT_ATTACHMENT_FEEDBACK_LOOP_OPTIMAL_EXT;
```

Comment:
If `layout` is any of `TRANSFER_SRC_OPTIMAL`, `TRANSFER_DST_OPTIMAL` or `GENERAL`, we need to return `layout` for this.